src/containers/CommunityHome/index.tsx

```diff
@@ -11,32 +10,304 @@ import {
+            </CardDescription>
+            <CardDescription className="mb-2">
+              <Strong> Decimals:</Strong> {community.token.decimals}
+            </CardDescription>
+            <Button
+              variant="outline"
+              onClick={() =>
+                onOpenAccount(community.scan.url, community.token.address)
+              }
+            >
+              View on {community.scan.name}
+            </Button>
+          </CardContent>
         </Card>
       }
+      TransactionTable={
+        <>
+          <Flex justify="between" align="center" className="py-4 mt-2">
+            <Select
+              value={paginationCount}
+              onValueChange={(value) => onFilterChange(value)}
+            >
+              <SelectTrigger className="w-[180px]">
+                <SelectValue placeholder="10" />
+              </SelectTrigger>
+              <SelectContent>
+                <SelectItem value="10">10</SelectItem>
+                <SelectItem value="25">25</SelectItem>
+                <SelectItem value="50">50</SelectItem>
+              </SelectContent>
+            </Select>
+            <Flex>
+              <Box className="mr-2 cursor-pointer">
+                <IconButton
+                  className="m-2"
+                  variant="soft"
+                  onClick={() => onRefresh()}
+                >
+                  <RotateCcw />
+                </IconButton>
+              </Box>
+              <Box>
+                <IconButton
+                  className="m-2 cursor-pointer"
+                  variant="soft"
+                  color="green"
+                >
+                  <Download />
+                </IconButton>
+              </Box>
+            </Flex>
+          </Flex>
+          {transfersLoading ? (
+            <Skeleton
+              className="mb-3"
+              style={{ height: 600, width: 900 }}
+            ></Skeleton>
+          ) : (
+            <Table>
+              <TableHeader>
+                <TableRow>
+                  <TableHead>Transfer Hash</TableHead>
+                  <TableHead>Transaction Hash</TableHead>
+                  <TableHead>Date</TableHead>
+                  <TableHead>From</TableHead>
+                  <TableHead>To</TableHead>
+                  <TableHead>Amount</TableHead>
+                  <TableHead>Description</TableHead>
+                  <TableHead>Status</TableHead>
+                </TableRow>
+              </TableHeader>
+              <TableBody>
+                {transfers &&
+                  transfers?.map((transaction: Transfer) => {
+                    return (
+                      <TableRow key={transaction.hash}>
+                        <TableCell>
+                          {shortenAddress(transaction?.hash)}
+                        </TableCell>
+                        <TableCell>
+                          {shortenAddress(transaction?.tx_hash)}
```

Comment:
Could you make this clickable, like a link?

Url format: `${community.scan.url}/tx/${transaction?.tx_hash}`

Reply:
Look at the table on this page: https://celoscan.io/token/0x3c960e72bbbd837293e75080e1d0fee6a4640357

Reply:
done